examples/feeds/update_flights_feed_item_string_attribute_value.py

```diff
@@ -0,0 +1,339 @@
+        )
+    except GoogleAdsException as ex:
+        print(
+            f'Request with ID "{ex.request_id}" failed with status '
+            f'"{ex.error.code().name}" and includes the following errors:'
+        )
+        for error in ex.failure.errors:
+            print(f'\tError with message "{error.message}".')
+            if error.location:
+                for field_path_element in error.location.field_path_elements:
+                    print(f"\t\tOn field: {field_path_element.field_name}")
+        sys.exit(1)
+
+
+def _flight_placeholder_fields_map(client, customer_id, feed_resource_name):
+    """Maps place holder fields and feed attributes for a flights feed.
+
+    See FlightPlaceholderField.php for all available placeholder field values.
+
+    Args:
+        client: an initialized GoogleAdsClient instance
+        customer_id: a client customer ID
+        feed_resource_name: a resource name for a Feed
+
+    Returns:
+        a dict mapping placeholder fields to feed attributes
+    """
+    flight_placeholder_field_enum = client.get_type(
+        "FlightPlaceholderFieldEnum", version="v6"
+    )
+
+    return _placeholder_field_maps(
+        client,
+        customer_id,
+        feed_resource_name,
+        {
+            "Flight Description": flight_placeholder_field_enum.FLIGHT_DESCRIPTION,
+            "Destination ID": flight_placeholder_field_enum.DESTINATION_ID,
+            "Flight Price": flight_placeholder_field_enum.FLIGHT_PRICE,
+            "Flight Sale Price": flight_placeholder_field_enum.FLIGHT_SALE_PRICE,
+            "Final URLs": flight_placeholder_field_enum.FINAL_URLS,
+        },
+    )
+
+
+def _placeholder_field_maps(
+    client, customer_id, feed_resource_name, feed_attribute_names_map
+):
+    """Retrieves the placeholder fields to feed attributes map for a feed.
+
+    The initial query retrieves the feed attributes, or columns, of the feed.
+    Each feed attribute will also include the feed attribute ID, which will be
+    used in a subsequent step.
+
+    Then a map is created for the feed attributes (columns) and returned:
+      - The keys are the placeholder types that the columns will be.
+      - The values are the feed attributes.
+
+    Args:
+        client: an initialized GoogleAdsClient instance
+        customer_id: a client customer ID
+        feed_resource_name: a resource name for a Feed
+        feed_attribute_names_map: the associative array mapping from feed
+            attribute names to placeholder fields
+
+    Returns:
+        a dict mapping placeholder fields to feed attributes
+    """
+    google_ads_service = client.get_service("GoogleAdsService", version="v6")
+    # Constructs the query to get the feed attributes for the specified feed
+    # resource name.
+    query = f"""
+        SELECT
+          feed.attributes
+        FROM feed
+        WHERE feed.resource_name = '{feed_resource_name}'
+        LIMIT 1"""
```

Comment:
why the LIMIT 1 given there should be only one value with that resource name?

Reply:
Good question, I'll take it out.